stdlib/REPL/Project.toml

```diff
@@ -7,8 +7,8 @@ Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
 Sockets = "6462fe0b-24de-5631-8697-dd941f90decc"
 
 [extras]
-Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
Probably best not to include this?

Reply:
well, I included this because `(REPL) pkg> resolve` automatically made this change, but I'm totally okay not to include this :)
Maybe this should be done in a separate PR or somewhere.